js/rank_states_utils.js

```diff
@@ -3,3 +3,142 @@ var svgStates = d3.select("#rank-states-interactive")
   .append("svg")
   .attr('viewBox', '0 0 1000 300')
   .attr('preserveAspectRatio', 'xMidYMid');
+
+var stateMap = svgStates.append('g')
+  .attr('id','ranked-states-map')
+  .attr('transform',"scale(0.4)");
+
+stateMap.append('g')
+  .attr('id','ranked-states-moved');
+
+stateMap.append('g')
+  .attr('id','ranked-states-dragable');
```

Comment:
just to be consistent since the classes spell this right, might want to add another `g` 😄